src/reducers/userReducer.js

```diff
@@ -0,0 +1,24 @@
+import { userActionTypesConstants } from "constants/userConstants";
+
+const initialState = {
+  loading: false,
+};
+
+const { USER_REQUEST, USER_REQUEST_SUCCESS } = userActionTypesConstants;
+
+export const user = (state = initialState, { type, data }) => {
+  switch (type) {
+    case USER_REQUEST:
+      return {
+        ...state,
+        loading: true,
+      };
+    case USER_REQUEST_SUCCESS:
+      return {
+        ...state,
+        loading: false,
+      };
```

Comment:
You could store User's data like profile information or other things here

Reply:
Are you saying to use this to store the user's data instead of sessionStorage?